backend/.env.test

```
@@ -1,51 +1,83 @@
-KAFKA_BOOTSTRAP_SERVERS=localhost:9092
-KAFKA_TOPIC_PREFIX=test.
-SCHEMA_SUBJECT_PREFIX=test.
-SCHEMA_REGISTRY_URL=http://localhost:8081
-
-# Reduce consumer pool and timeouts for faster test startup/teardown
-# https://github.com/aio-libs/aiokafka/issues/773
-SSE_CONSUMER_POOL_SIZE=1
-KAFKA_SESSION_TIMEOUT_MS=6000
-KAFKA_HEARTBEAT_INTERVAL_MS=2000
-KAFKA_REQUEST_TIMEOUT_MS=5000
+DATABASE_NAME=integr8scode_db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_NAME key should go before the PROJECT_NAME key

(UnorderedKey)